lms/static/js/student_account/views/RegisterView.js

```diff
@@ -127,6 +127,50 @@
                         jsHook: this.authWarningJsHook,
                         message: fullMsg
                     });
+                },
+
+                getFormData: function() {
+                    var obj = FormView.prototype.getFormData.apply(this, arguments),
+                        $form = this.$form,
+                        elements = $form[0].elements,
+                        i,
+                        len = elements.length,
+                        $el,
+                        $label,
+                        key = '',
+                        $email_element,
+                        $confirm_email_element,
+                        email = '',
+                        confirm_email = '';
+
+                    for (i = 0; i < len; i++) {
+                        $el = $(elements[i]);
+                        key = $el.attr('name') || false;
+
+                        if (key == 'email') {
+                            $email_element = $el;
+                        } else if (key == 'confirm_email') {
+                            $confirm_email_element = $el;
+                        }
+                    }
+
+                    if ($confirm_email_element) {
+                        email = $email_element.val();
+                        confirm_email = $confirm_email_element.val();
+                        $label = $form.find('label[for=' + $confirm_email_element.attr('id') + ']');
+
+                        if (email !== confirm_email) {
+                            this.errors.push($confirm_email_element.data('errormsg-required'));
```

Comment:
Note that other error messages show up on this form as `<li>` elements, so they get bullet points. Let's see how we're doing that for the other form messages and do that here as well. It might also be easier to do this validation and erroring in Python than in JS.